vue/components/atoms/thumbnail/thumbnail.vue

```diff
@@ -83,23 +88,52 @@ export const Thumbnail = {
     data: function() {
         return {
             image: null,
-            loaded: false
+            loaded: false,
+            hide: false
         };
     },
     computed: {
         active() {
             return this.frame === this.$store.state.currentFrame;
+        },
+        classes() {
+            const base = {
+                active: this.active,
+                loaded: this.loaded,
+                hide: this.hide
+            };
+            return base;
         }
     },
     methods: {
         showFrame() {
             this.$bus.trigger("show_frame", this.frame);
         },
+        setVideo() {
+            if (!this.frame.startsWith("video")) return;
+
+            const videoThumbnailURL = this.$ripe._getVideoThumbnailURL({
+                name: this.frame.split("video-")[1]
+            });
+            this.$refs.image.src = videoThumbnailURL;

```

Comment:
Because it would crash with the existing approach of `bindImage`, since the same `<img />` is used both by the bindImage (where the ref is used) and the video one